kit/src/elements/folder/mod.rs

```diff
@@ -84,7 +79,7 @@ pub fn Folder<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
                 },
                 with_rename.then(|| rsx! (
                     Input {
-                        disabled: *disabled,
+                        disabled: disabled,
```

Comment:
i'm surprised that `cargo fmt` didn't try to collapse this

Reply:
Usually in macros, rustfmt doesnt have much of an effect on restructuring (or if it does, i havent been seeing it). I also know due to `Input` being a prop instead of a struct, it may not collapse like it would with your usual structs fields (while it is expanding at compile time)